packages/common/src/stores/AccountController.ts

```diff
@@ -202,7 +202,7 @@ export default class AccountController {
 
   logout = async (logoutOptions: { includeNetworkRequest: boolean } = { includeNetworkRequest: true }) => {
     // this invalidates all entitlements caches which makes the useEntitlement hook to verify the entitlements.
-    await queryClient.invalidateQueries('entitlements');
+    await queryClient.invalidateQueries({ queryKey: ['entitlements'] });
```

Comment:
Is the explicit object needed or is `queryClient.invalidateQueries(['entitlements']);` also valid?

Reply:
In this version yes, in future version of tanstack query not anymore. I was tempted to update it, but for this reason I prefer to leave it.